### ISSUE TITLE
is there online sql running option like sqlime.org

### ISSUE THREAD

**Issue by Sandy4321** [cathytanimura/sql_book#1]
is there online sql running option like sqlime.org
http://sqlime.org/about.html

do you have second book for more complicated examples ?

**Comment by AKPFLOW** [comment]
Yes. You can run the SQL queries on https://www.sqlines.com/online

**Comment by Sandy4321** [comment]
great thanks
1
will it run for all sql functionalities like windows ?
2
how to upload data base files what you use for book?

**Comment by AKPFLOW** [comment]
Hi Sandy4321, 

I am not the author of the book. I am here to help.

The website I provided will help convert/transform the PostgreSQL to MSSQL or other desired format. 


2. I believe Cathy gave some instructions on how to upload the database files. 
You can check our stackflow for more resources.. 


Hope this is helpful

Regards

**Comment by AKPFLOW** [comment]
Hello Sandy4321,

I managed to set up PostgreSQL on my windows Pc and access it using Dbeaver.. This is super cool .

I give you updates as I run
